packages/css/src/page-menu/page-menu.scss

```
@@ -0,0 +1,73 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+@mixin reset-list {
+  box-sizing: border-box;
+  margin-block: 0;
+  padding-inline: 0;
+  -webkit-text-size-adjust: 100%;
+}
+
+@mixin reset-item {
+  box-sizing: border-box;
+  -webkit-text-size-adjust: 100%;
+}
+
+.amsterdam-page-menu__list {
+  align-items: center;
+  column-gap: var(--amsterdam-page-menu-list-column-gap);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  list-style: none;
+
+  @include reset-list;
+}
+
+@mixin page-menu-item {
+  color: var(--amsterdam-page-menu-item-color);
+  display: inline-flex;
+  flex-direction: row;
+  font-family: var(--amsterdam-page-menu-item-font-family);
+  font-size: var(--amsterdam-page-menu-item-narrow-font-size);
+  font-weight: var(--amsterdam-page-menu-item-narrow-font-weight);
+  gap: var(--amsterdam-page-menu-item-gap);
+  line-height: var(--amsterdam-page-menu-item-line-height);
+  text-align: center;
+  text-decoration: var(--amsterdam-page-menu-item-text-decoration);
+  touch-action: manipulation;
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-page-menu-item-wide-font-size);
+  }
+}
+
+.amsterdam-page-menu__link {
+  @include page-menu-item;
+  @include reset-item;
+}
+
+.amsterdam-page-menu__button {
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  margin-block: 0;
+  padding-inline: 0;
+
+  @include page-menu-item;
+  @include reset-item;
+}
+
+.amsterdam-page-menu__link a:hover,
+.amsterdam-page-menu__button button:hover {
```

This isn't correct anymore, right?

```suggestion
.amsterdam-page-menu__link:hover,
.amsterdam-page-menu__button:hover {
```